### PR TITLE
new logo name to resolve conflict and gruntfile.js updated

### PR DESCRIPTION
**Pull request by joonhocho**
@andrewconner

### CODE REVIEW

File: kifi-backend/modules/shoebox/public/Gruntfile.js [hunk 1]
```diff
@@ -196,7 +196,7 @@ module.exports = function (grunt) {
 				tasks: 'default'
 			},
 			statics: {
-				files: ['css/*.less', 'static/css/*.less'],
+				files: ['css/*.less', 'css/*.less'],
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/public/Gruntfile.js:199]: Redundant elements are redundant.

File: kifi-backend/modules/shoebox/public/static/static [hunk 2]
```diff
@@ -0,0 +1 @@
+.
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/public/static/static:1]: This symbolic link isn't needed anymore.

Comment (andrewconner) [inline kifi-backend/modules/shoebox/public/static/static:1]: And could break the build? Not sure how that worked before.

File: kifi-backend/modules/shoebox/public/Gruntfile.js (hunk 1 above)
Comment (joonhocho) [inline kifi-backend/modules/shoebox/public/Gruntfile.js:199]: lol.

File: kifi-backend/modules/shoebox/public/static/static (hunk 2 above)
Comment (joonhocho) [inline kifi-backend/modules/shoebox/public/static/static:1]: oh sorry

Comment (joonhocho) [inline kifi-backend/modules/shoebox/public/static/static:1]: oh .gitignore is removed so that is why they are added. I will remove them.

**Pull request merged**